modulo2/projeto-whatslab/src/App.js

```diff
@@ -0,0 +1,106 @@
+import './App.css';
+import React from 'react';
+import styled from 'styled-components';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+  border: 2px solid black;
+  background-color: lightgray;
+`
+const Mensagens = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  width: 50em;
+  height: 40em;
+  align-items: center;
+  border: 2px solid black;
+  background-color: white;
+`
+const EspacoInputs = styled.div`
+  display: flex;
+  width: 50em;
+  height: 2em;
+  justify-content: space-around;
+  border: 2px solid black;
+  background-color: yellowgreen;
+`
+
+const InputUsuario = styled.input`
+  display: flex;
+  flex-direction: column;
+  width: 10rem;
+`
+const InputMensagem = styled.input`
+  display: flex;
+  flex-direction: column;
+  width: 30rem;
+`
+
+class App extends React.Component {
+
+//IMPLEMENTAR A LÓGICA DOS INPUTS AQUI
+
+//Estado
+state = {
+  inputUsuario: "",
+  UsuarioFinal: "",
+  inputMensagem: "",
+  MensagemFinal: "",
+    
+};
+
+  
+pegarValorUsuario = (event) => {
+  console.log(event.target.value);
+  this.setState({
+  inputUsuario: event.target.value
+  });
+};
+
+pegarValorMensagem = (event) => {
+  console.log(event.target.value);
+  this.setState({
+  inputMensagem: event.target.value
+  });
+};  
+
+Enviando = () => {
+  this.setState ({UsuarioFinal: this.state.inputUsuario + ':'})
+  this.setState ({inputUsuario: ""})
+  this.setState ({MensagemFinal: this.state.inputMensagem})
+  this.setState ({inputMensagem: ""})
+  
```

Comment:
Quando alterar vários estados de uma só vez é recomendável usar somente 1 this.setState para cumpri essa tarefa:
```
this.setState ({
UsuarioFinal: this.state.inputUsuario + ':',
  inputUsuario: "",
  MensagemFinal: this.state.inputMensagem,
  inputMensagem: ""
})
```